loopy/translation_unit.py

```diff
@@ -195,7 +196,8 @@ class TranslationUnit:
     entrypoints: FrozenSet[str]
 
     def __post_init__(self):
-        assert isinstance(self.entrypoints, frozenset)
+
+        assert isinstance(self.entrypoints, abc_Set)
```

Comment:
Could you please explain why this is needed? Thanks!

Reply:
Sure, these changes are necessary for compatibility with alternative set implementations, in particular https://github.com/matthiasdiener/orderedsets.